_posts/2019-12-25-foreman-community-newsletter---december-2019.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Foreman Community Newsletter (December 2019)
+date: 2019-12-25 11:01:36
+author:
+ name: Ori Rabin
+ uri: orabin
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+ Lots of conferences coming up!
+---
+
+We have plenty of conferences next month and tickets are running out fast! 
+
+
+### 1.24 released!
+
+Foreman 1.24 is out with almost 250 features and bug fixes!
+This will be the last version before Foreman 2.0 so please notice all the deprecation
+warnings in the [release notes][1_24].
+
+As ever, many thanks to the contributors to this release - coders, RC testers,
+documentation writers, and bug/feature reporters. Foreman wouldn't be what it
+is without you! If you do find issues with, please do [report them!][issues]
+
+### Katello 3.14 and 3.13.3 are out!
+
+Katello [3.14][k_3_14] was released with an amazing number of 126 features and bug fixes!
+On the same day [3.13.3][k_3_13_3] was also released with 2 bug fixes.
+Huge thanks to all the contributors to this release - as with core, we couldn't
+put out these releases without you! Further issues with Katello should be
+[reported][k_issues], as always.
+
+
+[1_24]: https://community.theforeman.org/t/foreman-1-24-0-has-been-released/16545
+[k_3_14]: https://community.theforeman.org/t/katello-3-14-0-has-been-released/16566
+[k_3_13_3]: https://community.theforeman.org/t/katello-3-13-3-is-available/16565
+[issues]: https://projects.theforeman.org/issues
+[k_issues]: https://projects.theforeman.org/projects/katello/issues
+
+
+### Conferences
+
+[DevConf.CZ](https://www.devconf.info/cz/): 24-26 January in Brno.
+We'll be there at the Foreman stand.
+Tickets are running out fast so go and get one!
+
+[FOSDEM](https://fosdem.org/2020): 1-2 February in Brussels.
+Come visit us at the booth or drop by on one of the Foreman talks
+in the Infra room!
+
+[CfgMgmtCamp](https://cfgmgmtcamp.eu/ghent2020/): 3-5 in Ghent.
+See you there at the booth, Foreman room and our traditional
+hackday on the 5th! Tickets are available. 
+
+### [More Upcoming Events](https://community.theforeman.org/c/events/l/calendar)
+
+All Foreman events are available via [the forum
+calendar](https://community.theforeman.org/calendar) - hit the Subscribe button
+(which will give you an ICS link). You can also use the "Add To Calendar" button
+on specific events if you don't wish to import the whole thing.
+
```

Comment:
I didn't see any links from the calendar? perhaps it make sense to link from here to our calendar (or to the official documentation of  the events)...

Reply:
I added the events to the calendar, I can switch the links here as well.